auth/permisos.ts

```diff
@@ -190,6 +190,7 @@ export default [
         avoidAll: true,
         child: [
             { key: 'visualizacionHuds', title: 'Visualización HUDS por paciente', type: 'boolean' },
+            { key: 'impresionHuds', title: 'Imprimir HUDS por prestación', type: 'boolean', visibility: 'restricted' },
```

Comment:
La palabra _Huds_ esta de más, porque ya esta dentro de **HUDS**
```suggestion
            { key: 'impresion', title: 'Imprimir cualquier prestación', type: 'boolean', visibility: 'restricted' },
```